safe/impact_function/multi_exposure_wrapper.py

```diff
@@ -443,3 +467,55 @@ def run(self):
             self._analysis_summary, analysis_width, analysis_color)
 
         return ANALYSIS_SUCCESS, None
+
+    def generate_report(self, components, output_folder=None, IFACE=None):
```

Comment:
A variable must be lowercase. See here where I asked your review: https://github.com/inasafe/inasafe/pull/4581